docs/utilities.md

```diff
@@ -0,0 +1,67 @@
+# Utilities
+
+
+::: sqlalchemy_history.utils
+
+
+## changeset
```

Comment:
Some notes after review:
 - Does `::` work in mkdocs ? That is normally a rst syntax
 - `issued\_at` probably does not need the backslash in markdown
 - Use a markdown formatter like prettier ?
 - Avoid REPL based python code snippet to make it easier to copy code (unless we are trying to show an example with outputs)
 - Some REPL examples have indentation errors